esx_service/utils/auth.py

```diff
@@ -91,34 +91,34 @@ def get_default_tenant():
         return None, None, None
 
 
-def get_default_privileges():
-    """ Get DEFAULT privileges by querying the auth DB.
-        DEFAULT privilege will match any (tenant_uuid, datastore) pair which
-        does not have an entry in privileges table explicitly
+def get_all_ds_privileges(tenant_uuid):
+    """ Get privileges on _ALL_DS by querying the auth DB.
+        For the tenant with given tenant_uuid, this privilege
+        will match any datastore which does not have an entry
+        in privileges table explicitly
```

Comment:
This **is** confusing and also  sounds different from what we've discussed. Can you elaborate on what do you think admin wants to do here? Is it the old "default privilege" fallback for all ? 
Maybe "Get  _ALL_DS priviledge from AuthDB. This privilege is 'catch-all' fallbacks for datastores not explicitly bound to a tenant_uuid' I am guessing here, I do not really understand he comment